airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_events.py

```diff
@@ -120,19 +120,19 @@ def test_given_start_date_before_30_days_stripe_limit_and_slice_range_when_read_
         slice_range = timedelta(days=30)
         slice_datetime = start_date + slice_range
         http_mocker.get(  # this first request has both gte and lte before 30 days even though we know there should not be records returned
-            _a_request().with_created_gte(start_date).with_created_lte(slice_datetime).with_limit(100).build(),
+            _a_request().with_created_gte(start_date).with_created_lte(slice_datetime - timedelta(days=1)).with_limit(100).build(),
```

Comment:
It feels like it should be only 1 second here and not 1 day. This probably relates to my comment about cursor_granularity

Reply:
Yes, updated to 1 second